hibernate-search-plugins/org.jboss.tools.hibernate.search.runtime.v_4_0/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Yes, it's normal. Look for example at jbosstools-openshift: https://github.com/jbosstools/jbosstools-openshift/tree/master/plugins/org.jboss.tools.openshift.client. You'll find there `.classpath` and `.project`. But i think it's better to ask @koentsje for the explanation